views/partials/dashboards/breaches-dash.hbs

```diff
@@ -37,9 +37,24 @@
       {{> dashboards/manage-email-link variableClass="hide-mobile"}}
     </div>
     <div class="email-cards flx flx-col jst-cntr">
-      {{#each this.verifiedEmails}}
+      {{#eachFromTo this.verifiedEmails 0 1 }}
         {{> email-card }}
-      {{/each}}
+      {{/eachFromTo}}
+      {{#if ./../supportedLocalesIncludeEnglish}}
+        <div class="private-relay-dashboard-card">
+          <a href="/protect-my-email" class="private-relay-dashboard-card-border flx private-relay-cta" data-analytics-label="join-the-waitlist-link" data-user-is-signed-up={{ ./../userHasSignedUpForRelay }}>
+            <div class="private-relay-dashboard-card-content flx">
+              <div class="private-relay-dashboard-card-copy">
+                <p class="bold ff-Met private-relay-dashboard-card-subhead">Clear inbox clutter. Keep your personal email private.</p>
```

Comment:
Is it standard practice for the default (EN-US) strings to already be in the element, or should they be placed dynamically from the localization render?

Reply:
Definitely not standard practice but these strings are not being localized unfortunately and will be removed once the test is over anyway.